### PR TITLE
subpar:0.1.0

### PR DESCRIPTION
**Pull request by tingerrr**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: Provides easy to use sub figure environments.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

### CODE REVIEW

**Comment by tingerrr** [comment]
I have marked this as a draft to wait for the fixes in #587.

**Comment by tingerrr** [comment]
I have upstreamed the fixes and updated the generated manual.

File: packages/preview/subpar/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+name = "subpar"
+version = "0.1.0"
+compiler = "0.11.0"
+authors = ["tinger <me@tinger.dev>"]
+repository = "https://github.com/tingerrr/subpar"
+description = "Create sub figures easily."
+categories = ["components", "model"]
+keywords = ["figures", "sub"]
+license = "MIT"
+exclude = ["examples", "doc", "README.md"]
```
Comment (elegaanz) [inline packages/preview/subpar/0.1.0/typst.toml:12]: Your README should not be excluded I think.

**Review by elegaanz — COMMENTED** [review]
Thanks! Just a small remark, otherwise it looks fine.

File: packages/preview/subpar/0.1.0/typst.toml (hunk 1 above)
Comment (tingerrr) [reply packages/preview/subpar/0.1.0/typst.toml:12]: Wasn't much of an issue before, as people usually check either the universe page or the GitHub repo out, but I'll include it.

**Comment by tingerrr** [comment]
PS: make sure to use squash and merge.

**Comment by elegaanz** [comment]
Thank you :)

**Pull request merged**